ansible/ansible.cfg

```diff
@@ -1,5 +1,6 @@
 [defaults]
 retry_files_enabled = False
+remote_tmp=/tmp
```

Comment:
If everything is moved under the `installers/pgo-installer` folder, should this go there?

Reply:
This is required in the `$PGOROOT/ansible` roles to allow installing the operator from within a kubecluster. The new roles `$PGOROOT/installers/pgo-installers/ansible` setup and run the install job. The original roles still install the operator. Would renaming one of the ansible directories help make this clear?

Reply:
I guess my question was more fundamental, i.e. should we move the `ansible` folder out of `$PGOROOT` into a different directory?

Reply:
That would make sense to me. Possibly move `$PGOROOT/ansible` to `$PGOROOT/installers/ansible`

Reply:
:+1: